spylon_kernel/init_spark_magic.py

```diff
@@ -31,9 +31,15 @@ def __init__(self, kernel):
         self.env['launcher'] = spylon.spark.launcher.SparkConfiguration()
         self.log = logging.Logger(self.__class__.__name__)
 
-    def cell_init_spark(self):
-        """Starts a SparkContext with a custom configuration defined
-        using Python code.
+    # Use argparse to parse the whitespace delimited cell magic options
+    # just as we would parse a command line.
+    @option(
```

Comment:
optparse apparently. Will adjust the comment.

https://github.com/Calysto/metakernel/blob/d100ea4a101ed3f385ce0a5bf1592741125f7e29/metakernel/magic.py#L15